libcxx/include/string

```diff
@@ -1630,11 +1630,11 @@ public:
 
   _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX20 const value_type* c_str() const _NOEXCEPT { return data(); }
   _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX20 const value_type* data() const _NOEXCEPT {
-    return std::__to_address(__get_pointer());
+    return std::addressof(__get_pointer());
```

Comment:
The current change doesn't seem right. `__get_pointer` already returns a pointer prvalue, and thus we should use it directly and cannot use `addreessof` on it.